### PR TITLE
Add transitions for items in the Toggle Group Component

### PR DESCRIPTION
**Pull request by razwan**
Add transitions for items in toggle group used in cards collection inspector controls

### CODE REVIEW

File: src/components/toggle-group/_editor.scss [hunk 1]
```diff
@@ -1,12 +1,20 @@
+.components-toggle-group {
+	overflow: hidden;
```
Comment (madalingorbanescu) [inline src/components/toggle-group/_editor.scss:2]: I think the animation would look better with with `overflow: visible`. Now the item it looks like is cut-off.
This is how it looks with `overflow: hidden`: https://share.getcloudapp.com/yAu27wmp
And this is how it looks with `overflow: visible`: https://share.getcloudapp.com/jkuld9GP

Comment (georgeolaru) [reply src/components/toggle-group/_editor.scss:2]: @madalingorbanescu I'm not seeing much a difference as there is a "jump" throughout the recording video – is that caused by the overflow or merely by scroll?

Comment (madalingorbanescu) [reply src/components/toggle-group/_editor.scss:2]: That jump was caused by the scroll.
Here is another videos, where the difference is more visible:
With `overflow: hidden`: https://share.getcloudapp.com/eDu6QmkO
With `overflow: visible`: https://share.getcloudapp.com/yAu27pXW

When an element is removed, and added in **Elements you aren't using**, the right border on that element is not visible until the transition ends.

File: src/components/toggle-group/_editor.scss [hunk 2]
```diff
@@ -1,12 +1,20 @@
...
+}
+
+.components-toggle-group__toggle-list {
```
Comment (madalingorbanescu) [inline src/components/toggle-group/_editor.scss:5]: On animated elements from toggle-group we should add `will-change: opacity, transform` to improve performance.

File: src/components/toggle-group/_editor.scss (hunk 1 above)
Comment (georgeolaru) [reply src/components/toggle-group/_editor.scss:2]: @madalingorbanescu I get it now – good catch! 👏 

Small but nice improvement – it seems like we're polishing a diamond here 💎😊

**Pull request merged**